shared/cfg/base.cfg

```diff
@@ -620,6 +624,22 @@ migrate_proto = "tcp"
 # Remember to open ports on both source and destination
 migrate_port = 49152
 
+##### migration_template use only parameters
+
+# Whether to migrate vm back to src host, default to yes 
```

Comment:
How about remove the word "default to yes". I want it default to "yes" in module migration_template.py, because in cross feature testing, we usually need to migrate vm back

Reply:
@fangge1212 , this base.cfg is for all python files, not only one file. So if you want it to be 'yes', you'd better overwrite it in migration_template.cfg.

Reply:
I have updated it to no